src/types/customer/index.ts

```diff
@@ -12,8 +12,8 @@ export interface ICustomer {
   name: string;
   email: string;
   locale: Locale;
-  recentlyUsedMethods: Array<PaymentMethod>;
-  metadata: any;
+  recentlyUsedMethods: PaymentMethod[];
+  metadata: Record<string, string>;
```

Comment:
is this true? `metadata: Record<string, string>;`  ?

Reply:
Yes, this is correct. It can be a JSON object with any key / value pair.